dockstore-client/pom.xml

```diff
@@ -278,6 +278,12 @@
             <groupId>org.glassfish.jersey.media</groupId>
             <artifactId>jersey-media-multipart</artifactId>
         </dependency>
+
+        <dependency>
```

Comment:
Hmmm, no releases since 2017, open pull requests from 2018
Will allow it for now, but add a code comment with the above and that we should probably look into something else long term.

Reply:
Added a note for details on the repo. https://github.com/dockstore/dockstore-cli/pull/85/commits/40e312e1e70af4671467e15dd14c96b57dea52e1

An important side note is that this repo does not support temporary AWS credentials (session credentials), and so we'd need to role our own modifications for that at some point anyways.

 That being said, we can also use the AWS Java SDK to help calculate these signatures (https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/auth/AWS4Signer.html), and it would look something like the code comment here https://ucsc-cgl.atlassian.net/browse/SEAB-3284. 

The SDK is heavier weight and does some behind-the-scenes header additions which makes calculating the signature for the Invocation.Builder a little more complicated (didn't dive too much into this, so might be easier than expected), but also has the benefit of being actively supported.

Long-winded way of saying that I'm not quite sure which is the correct approach.